README.md

```diff
@@ -1,20 +1,14 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
-in a simple executable script/binary.
+Command-line tool and programmatic Python access to the Conjur API to manage your Conjur resources. 
 
-[![Test Coverage](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/test_coverage)](https://codeclimate.com/github/cyberark/conjur-api-python3/test_coverage) [![Maintainability](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/maintainability)](https://codeclimate.com/github/cyberark/conjur-api-python3/maintainability)
-
----
+This repository includes both the self-contained Conjur command-line tool (`conjur`) and Python3-based SDK for Conjur.
```

Comment:
I like this sentence much more :) but it feels a bit of repetition of the first one, dont you think?

Reply:
what about "This repository includes both the self-contained Conjur command-line tool (`conjur`) and Python3-based SDK for access to the Conjur API to manage your Conjur resources."